### PR TITLE
minor improvements for Slider

### PR DESCRIPTION
**Pull request by onemonster**

### CODE REVIEW

File: pages/deposit/[symbol].tsx [hunk 1]
```diff
@@ -144,7 +144,7 @@ const Deposit: NextPage<
                     <ClientComponent>
                       <Slider
                         key={'deposit-slider'}
-                        count={proceedsByEpochsDeposited.length}
+                        length={proceedsByEpochsDeposited.length}
```
Comment (graykode) [inline pages/deposit/[symbol].tsx:147]:
```suggestion
                        length={proceedsByEpochsDeposited?.length || 0}
```

Comment (onemonster) [reply pages/deposit/[symbol].tsx:147]: Since `initialData: []`, proceedsByEpochsDeposited is never null

**Review by graykode — COMMENTED** [review]
image
Can you check this error?

File: pages/deposit/[symbol].tsx (hunk 1 above)
Comment (onemonster) [reply pages/deposit/[symbol].tsx:147]: I'll do so in a separate PR

**Review by graykode — APPROVED** [review]

**Pull request merged**